airbyte-integrations/connectors/source-personio/source_personio/schemas/employees.json

```diff
@@ -0,0 +1,85 @@
+{
+  "type": "object",
+  "properties": {
+      "id": {
+        "type": ["null", "integer"]
+      },
+      "first_name": {
+        "type": ["null", "string"]
+      },
+      "last_name": {
+        "type": ["null", "string"]
+      },
+      "email": {
+        "type": ["null", "string"]
+      },
+      "status": {
+        "type": ["null", "string"]
+      },
+      "position": {
+        "type": ["null", "string"]
+      },
+      "supervisor": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": ["null", "integer"]
+          },
+          "first_name": {
+            "type": ["null", "string"]
+          },
+          "last_name": {
+            "type": ["null", "string"]
+          },
+          "email": {
+            "type": ["null", "string"]
+          }
+        }
+      },
+      "created_at": {
+        "type": ["null", "string"],
+        "format": "date-time"
+      },
+      "last_modified_at": {
+        "type": ["null", "string"],
+        "format": "date-time"
+      },
+      "office": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": ["null", "integer"]
+          },
+          "name": {
+            "type": ["null", "string"]
+          }
+        }
+      },
+      "department": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": ["null", "integer"]
+          },
+          "name": {
+            "type": ["null", "string"]
+          }
+        }
+      },
+      "team": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": ["null", "integer"]
+          },
+          "name": {
+            "type": ["null", "string"]
+          }
+        }
+      },
+      "dynamic_5333434": {
```

Comment:
What is this "dynamic_.."?
Is it a common column or something tenant specific?

Reply:
The naming is coming from personio's endpoint `company/employees`. 
It's basically a custom column that people and culture created on our personio (this column specifically is the "experience").